target_chains/ton/contracts/contracts/common/errors.fc

```diff
@@ -20,27 +20,28 @@ const int ERROR_INVALID_GUARDIAN_SET_UPGRADE_LENGTH = 1015;
 const int ERROR_INVALID_GOVERNANCE_CHAIN = 1016;
 const int ERROR_INVALID_GOVERNANCE_CONTRACT = 1017;
 const int ERROR_INVALID_GUARDIAN_ADDRESS = 1018;
+const int ERROR_SIGNATURE_INDEX_NOT_INCREASING = 1019;
 
 ;; Pyth
-const int ERROR_PRICE_FEED_NOT_FOUND = 1019;
-const int ERROR_OUTDATED_PRICE = 1020;
-const int ERROR_INVALID_MAGIC = 1021;
-const int ERROR_INVALID_MAJOR_VERSION = 1022;
-const int ERROR_INVALID_MINOR_VERSION = 1023;
-const int ERROR_UPDATE_DATA_SOURCE_NOT_FOUND = 1024;
-const int ERROR_INVALID_UPDATE_DATA_SOURCE = 1025;
-const int ERROR_DIGEST_MISMATCH = 1026;
-const int ERROR_INVALID_UPDATE_DATA_LENGTH = 1027;
-const int ERROR_INVALID_UPDATE_DATA_TYPE = 1028;
-const int ERROR_INVALID_MESSAGE_TYPE = 1029;
-const int ERROR_INSUFFICIENT_FEE = 1030;
-const int ERROR_INVALID_PROOF_SIZE = 1031;
-const int ERROR_INVALID_GOVERNANCE_DATA_SOURCE = 1032;
-const int ERROR_OLD_GOVERNANCE_MESSAGE = 1033;
-const int ERROR_INVALID_GOVERNANCE_TARGET = 1034;
-const int ERROR_INVALID_GOVERNANCE_MAGIC = 1035;
-const int ERROR_INVALID_GOVERNANCE_MODULE = 1036;
-const int ERROR_INVALID_CODE_HASH = 1037;
+const int ERROR_PRICE_FEED_NOT_FOUND = 2000;
+const int ERROR_OUTDATED_PRICE = 2001;
+const int ERROR_INVALID_MAGIC = 2002;
+const int ERROR_INVALID_MAJOR_VERSION = 2003;
+const int ERROR_INVALID_MINOR_VERSION = 2004;
+const int ERROR_UPDATE_DATA_SOURCE_NOT_FOUND = 2005;
+const int ERROR_INVALID_UPDATE_DATA_SOURCE = 2006;
+const int ERROR_DIGEST_MISMATCH = 2007;
+const int ERROR_INVALID_UPDATE_DATA_LENGTH = 2008;
+const int ERROR_INVALID_UPDATE_DATA_TYPE = 2009;
+const int ERROR_INVALID_MESSAGE_TYPE = 2010;
+const int ERROR_INSUFFICIENT_FEE = 2011;
+const int ERROR_INVALID_PROOF_SIZE = 2012;
+const int ERROR_INVALID_GOVERNANCE_DATA_SOURCE = 2013;
+const int ERROR_OLD_GOVERNANCE_MESSAGE = 2014;
+const int ERROR_INVALID_GOVERNANCE_TARGET = 2015;
+const int ERROR_INVALID_GOVERNANCE_MAGIC = 2016;
+const int ERROR_INVALID_GOVERNANCE_MODULE = 2017;
+const int ERROR_INVALID_CODE_HASH = 2018;
```

Comment:
changed these to 2xxx so that if we add new errors above then we dont have to change these again